src/SourceGenerators/ThisAssembly.Constants/readme.md

```diff
@@ -0,0 +1,28 @@
+**C# 9.0 ONLY**
```

Comment:
Should perhaps change to `C# 9.0 or later` now that we have 10 and soon 11 :)

Reply:
The C# version used in a project is totally unrelated to the ability to use source generators. It's the version of Roslyn that counts: the latest Roslyn will happily use incremental generators in a C# 1.0 project, or even in a VB project.

I'm going to update descriptions and readmes very soon.